be/src/exec/vectorized/hdfs_scanner_parquet.cpp

```diff
@@ -0,0 +1,78 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
```

Comment:
Should rebase the main and change it to 
// This file is licensed under the Elastic License 2.0. Copyright 2021-present StarRocks Limited.

Reply:
OK